TreePiece.cpp

```diff
@@ -3749,20 +3749,26 @@ void TreePiece::finishBucket(int iBucket) {
       }
 
 #if defined CUDA
```

Comment:
Can't this ifdef now go away?

Reply:
Doesn't this need to stay as is? If building with CUDA and bUseCpu is false, we want to make sure continueWrapUp() doesn't trigger.